layouts/partials/sections/about.html

```diff
@@ -3,11 +3,16 @@
   {{ $sectionID = .section.id }}
 {{ end }}
 
+{{ $author:= site.Data.author }}
```

Comment:
This bit seems to be copied around a bit, but i'm unclear when this value would be set. 
My understanding is that this map follows this file format
![image](https://user-images.githubusercontent.com/8029578/96675194-66e03c80-1338-11eb-884d-ab99b5833dbf.png)
Is it a possible convention to have an author file not in a locale folder?

Reply:
>Is it a possible convention to have an author file not in a locale folder?

This is done for multi-language support.

Reply:
yeah, sorry I mean I understand that `(index site.Data site.Language.Lang).author` accesses the particular `data/:locale/author.yml` file. But i'm not clear on when `site.Data.author` would ever be not-nil. My assumption is that 
`{{ $author:= site.Data.author }}` always initializes the `$author` variable to nil and then its overridden in the if block below. 

I guess i can't just do 
```go
{{ if (index site.Data site.Language.Lang).author }}
  {{ $author := (index site.Data site.Language.Lang).author }}
{{ end }}
```
because go's variable scoping means the `$author` variable isn't declared outside the if. Can you explain if `{{ $author:= site.Data.author }}` ever is set to not nil and if so what config it uses? I'd like to understand your config.yml, site.yml, and locale specific fallback logic a little better to help address some of the issues. Is the thought that someone could make a file like `data/author.yml` which would make `site.Data.author` use those values? 

Thanks!